backend/src/packages/auth/auth.service.ts

```diff
@@ -8,23 +12,42 @@ import {
 } from '~/packages/users/libs/types/types.js';
 import { type UserService } from '~/packages/users/user.service.js';
 
+import { type UserPrivateData } from '../users/libs/types/types.js';
+
 class AuthService {
   private userService: UserService;
 
-  public constructor(userService: UserService) {
+  private encrypt: IEncrypt;
+
+  public constructor(userService: UserService, encrypt: IEncrypt) {
     this.userService = userService;
+    this.encrypt = encrypt;
   }
 
   public async signIn(
     userSignInDto: UserSignInRequestDto,
   ): Promise<UserSignInResponseDto> {
-    const { email } = userSignInDto;
+    const { email, password } = userSignInDto;
 
     const user = await this.userService.findByEmail(email);
-
     if (!user) {
       throw new UserNotFoundError();
     }
+
+    const userPrivateData = (await this.userService.findPrivateData(
+      user.id,
+    )) as UserPrivateData;
```

Comment:
Would it better to use destructuring here?

Reply:
I believe that it won't be useful a lot. This statement isn't too complex, is it?

Reply:
Let's keep it as it is - like this we're grouping what values is internal, not from request payload